src/pages/Countries.js

```diff
@@ -1,37 +1,45 @@
 import React, { useEffect } from 'react';
 import { useSelector, useDispatch } from 'react-redux';
 import { Link } from 'react-router-dom';
-import { showCountries, selectCountry } from '../redux/Home/home';
+import { showCountries, selectCountry, showCountriesAll } from '../redux/Home/home';
 
```

Comment:
- [ ] OPTIONAL
The styling of the list on your Homepage does not conform to the alternating dual-color pattern of the provided [design template](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)). Kindly ensure that each cell of the list item should have surrounding cells having **different (alternating) background color(s)**.

**Your design:**
![image](https://user-images.githubusercontent.com/12745474/194685220-aca76cc9-cb12-4439-9e45-16db5773d439.png)


**The original design:**
![image](https://user-images.githubusercontent.com/12745474/142270463-4d16b9d3-7557-430c-80eb-a7c40589a4a6.png)